main.go

```diff
@@ -170,6 +174,15 @@ func run(cmd *cobra.Command, _ []string) {
 		managerOptions.Namespace = watchNamespace
 	}
 
+	if observabilityServiceEndpoint != "" {
+		tp, err := trace.CreateTraceProvider(observabilityServiceEndpoint, version)
```

Comment:
We may need to [set the logger](https://pkg.go.dev/go.opentelemetry.io/otel#SetLogger) here to ensure structured log messages according to `log-mode`.

```suggestion
		otel.SetLogger(logger)
		tp, err := trace.CreateTraceProvider(observabilityServiceEndpoint, version)
```